homeassistant/components/elevenlabs/__init__.py

```diff
@@ -20,12 +21,17 @@
 
 from .const import CONF_MODEL
 
+_LOGGER = logging.getLogger(__name__)
 PLATFORMS: list[Platform] = [Platform.TTS]
 
 
 async def get_model_by_id(client: AsyncElevenLabs, model_id: str) -> Model | None:
     """Get ElevenLabs model from their API by the model_id."""
-    models = await client.models.get_all()
+    try:
+        models = await client.models.list()
+    except ApiError as exc:
+        _LOGGER.error("Failure retrieving models: %s", exc)
```

Comment:
Why add extra logging? Wouldn't that already happen whereever this exception is handled.